src/metabase/api/permission_graph.clj

```diff
@@ -55,14 +55,14 @@
 
 (def ^:private TablePerms
   [:or
-   [:enum :all :segmented :none :full :limited :unrestricted :sandboxed :query-builder :no]
+   [:enum :all :segmented :none :full :limited :unrestricted :legacy-no-self-service :sandboxed :query-builder :no]
```

Comment:
Fixes graph parsing if it includes `legacy-no-self-service` so that it can be set via API in e2e tests